go/vt/schemadiff/table.go

```diff
@@ -231,6 +246,11 @@ func (d *RenameTableEntityDiff) IsEmpty() bool {
 	return d.Statement() == nil
 }
 
+// EntityName implements EntityDiff
+func (d *RenameTableEntityDiff) EntityName() string {
+	return d.from.Name()
```

Comment:
Is this deliberately the `from` name? Or should it be the `to` name? Or does it not matter here?

Reply:
While this isn't strictly in-use right now, it is deliberately `from` name. This is because the diff operates on the `from` table -- so that table is the one being affected by the operation. This is in line with the rest of diff types (with the exception of `CREATE` where the affected table is `to`).